app/views/external_users/claims/_agfs_hardship_claim_sidebar.html.haml

```diff
@@ -0,0 +1,25 @@
+%dl
+  %dt.fx-seed-gradFees{ data: { seed: claim.raw_basic_fees_total, autovat: @claim.apply_vat? ? 'true' : 'false' } }
+    = 'Hardship fees'
```

Comment:
Minor: Add all static content to locales file

Reply:
This is also the case in all other side bar partials. 

separate PR to tidy them all up with a consistent i18n pattern?